transfer.js

```diff
@@ -7,20 +7,14 @@ const accountPinElem = document.getElementById("account-pin");
 const currentUser = getUserByAccountNumber(currentUserAccountNumber);
 const allUsers = getAllUsers();
 // populate users in our database into the select field and remove the current user account number
-allUsers.forEach((user) => {
-  for (let i = 0; i < selectBeneficiaryAccountNumberElem.length; i++) {
-    if (
-      selectBeneficiaryAccountNumberElem.options[i].value ==
-      currentUserAccountNumber
-    ) {
-      selectBeneficiaryAccountNumberElem.remove(i);
-    }
-  }
-  const optionElem = document.createElement("option");
-  optionElem.value = user.accountNumber;
-  optionElem.textContent = `${user.accountNumber}`;
-  selectBeneficiaryAccountNumberElem.append(optionElem);
-});
+allUsers
+  .filter((user) => user.accountNumber !== currentUserAccountNumber)
+  .forEach((user) => {
+    const optionElem = document.createElement("option");
+    optionElem.value = user.accountNumber;
+    optionElem.textContent = `${user.accountNumber}`;
```

Comment:
The option text should be the user's name, not account number.

<img width="841" alt="Screenshot at Feb 16 12-58-45 PM" src="https://user-images.githubusercontent.com/51515797/219371463-d436ac34-9ef1-44e3-8937-1b43e96b6281.png">

Change to:
`optionElem.textContent = user.accountName;`

Reply:
Done